packages/cli/bin/cli.js

```diff
@@ -74,6 +81,17 @@ const argv = yargs
     describe: 'set log level to debug',
     type: 'boolean',
   })
+  .check(argv => {
```

Comment:
This check will run for every command. Is there a way to determine what the command name is in here? I'm thinking we could have some sort of filter that will let us only run this check for certain commands like "upload" and "watch". That way we don't run it on irrelevant commands.

Reply:
@brandenrodgers, working on incorporating your feedback now. Do you have a specific list of commands we'd like to run the check for? I'm also going to Slack you, but wanted to leave this here for posterity.

Reply:
Resolved: We've decided to check only the "upload" and "watch" commands for now.